docs/src/main/sphinx/connector/bigquery.rst

```diff
@@ -133,6 +133,8 @@ Property                                              Description
                                                       ``BIGNUMERIC`` and ``TIMESTAMP`` types are unsupported.        ``false``
 ``bigquery.views-cache-ttl``                          Duration for which the materialization of a view will be       ``15m``
                                                       cached and reused. Set to ``0ms`` to disable the cache.
+``bigquery.metadata-cache-ttl``                       Duration for which metadata retrieved from BigQuery            ``0ms``
```

Comment:
Rename to `bigquery.metadata.cache-ttl` for consistency with other connectors.

It's unfortunate that all connectors except JDBC use a prefix of connector name even though properties are already namespaced per connector.